package.json

```diff
@@ -84,6 +84,8 @@
     "**/@types/angular": "^1.6.56",
     "**/@types/hoist-non-react-statics": "^3.3.1",
     "**/typescript": "3.7.2",
+    "**/apollo-link": "^1.2.13",
+    "**/graphql-tools": "^4.0.7",
```

Comment:
Why are these necessary? We should avoid using these if at all possible.

Reply:
Removed